magma/syntax/combinational.py

```diff
@@ -275,7 +275,6 @@ def wrapped(fn):
     else:
         wrapped_combinational = ast_utils.inspect_enclosing_env(
                 _combinational,
-                decorators=[combinational],
-                st=env
+                decorators=[combinational]
```

Comment:
Can you try updating to the latest version of `ast_tools` and reverting this change? (`pip install --upgrade ast_tools`), the `st` argument was introduced in the latest release.

Reply:
Ok I guess I didn't realize I had to run pip upgrade, but that makes sense.